src/lib/docker-containers.mjs

```diff
@@ -392,10 +388,19 @@ async function waitForHealthCheck({
     });
   }
 
-  logger.stderr(
+  stderr(
     `[HealthCheck] Max attempts reached. Service at ${url} did not respond.`,
   );
   throw new CommandError(
     `[HealthCheck] Fauna at ${url} is not ready after ${maxAttempts} attempts. Consider increasing --interval or --maxAttempts.`,
   );
 }
+
+/**
+ * Outputs to stderr.
+ * @param {string} log The log
+ */
+function stderr(log) {
```

Comment:
Can we avoid the mutable global `color` by accepting it as an argument in `stderr`?

Reply:
then i would have to pass color to every private method in this module.

I'd prefer to memoize it.

Reply:
Understood. I realize I prefer verbosity over pit traps like global variables. You could curry the color `const stderrColored = stderr.bind(null, color)` if you want, but you probably don't 😜

Reply:
no that's dumb in this case ignore that.

But you could declare stderr function within `ensureContainerRunning` to close over the color provided.

```js
export async function ensureContainerRunning({
  containerName,
  hostIp,
  hostPort,
  containerPort,
  pull,
  interval,
  maxAttempts,
  color,
}) {
  function stderr(log) {  
    const logger = container.resolve("logger");
    logger.stderr(colorize(log, { format: Format.LOG, color }));
  }

  // ...
```

Reply:
then i'd have to pass that around.

Reply:
misread the diff, it's needed in several functions 👍🏻